File: 1. Two Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        found = False

        for i in range(len(nums)):
            for j in range(len(nums) - i):
                if nums[i] + nums[i + j] == target:
                    if i != j + i:
                        found = True
                        break
            if found:
                break
        return [i, j + i]
    

File: 1. Two Sum.php
class Solution {

/**
 * @param Integer[] $nums
 * @param Integer $target
 * @return Integer[]
 */
function twoSum($nums, $target) {

    // get arraylen of $nums
    $n = count($nums);

    for ($i = 0; $i < $n; $i++) {
        for ($j = $i + 1; $j < $n; $j++) {
            if ($nums[$j] + $nums[$i] == $target) {
                return array($i,$j);
            }
        }
    }
}
}
